src/components/picture/index.test.jsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from "@testing-library/react";
+
+import Picture from ".";
+import Source from "./_children/source";
```

Comment:
You should never consume the children components directly. This should be consumed via `Picture.Source`

Reply:
Hi @matthewroach,
Wanted to confirm this as we seem to be doing this with `Price`. 
https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.0.1/src/components/price/index.jsx#L3-L4
The only difference is that with `Price`, the files in the `./_children` folder are init-capped.
Please let me know if I'm misunderstanding. Thanks!

Reply:
The comment here relates to the test file, you are referencing a component file with you comment

Reply:
Yes, sorry, this response was meant for this comment. 
https://github.com/WPMedia/arc-themes-components/pull/127/files#r935474353

Reply:
And sorry, that comment is for another request. Thanks! This has been removed.